pyfr/integrators/base.py

```diff
@@ -192,6 +176,35 @@ def _get_axnpby_kerns(self, *rs, subdims=None):
 
         return kerns
 
+    @memoize
+    def _get_errest_kerns(self, *rs, norm):
+        kerns = proxylist([])
```

Comment:
Do we need a proxylist?

Reply:
Yes, otherwise this complains;
https://github.com/PyFR/PyFR/pull/208/files#diff-13a5ea96c99c7ef596846e0d1eaeafb8de4ec78acf33f9dc9de832fe5efb5ff6R29

Reply:
I'd be inclined to rewrite the code which uses the return values to avoid the need for them to be a proxylist.  (I am gradually trying to reduce the usage of the class.)